packages/arb-token-bridge-ui/src/components/common/HeaderNetworkInformation.tsx

```diff
@@ -28,7 +28,7 @@ export function HeaderNetworkInformation() {
         )}
       >
         <Image
-          src={getNetworkLogo(chain.id)}
+          src={getNetworkLogo(chain.id, 'light')}
```

Comment:
i'd use `invert` in the className list instead
https://tailwindcss.com/docs/invert

Reply:
Answer before